### PR TITLE
Add environment light intensity in GLSL

### PR DESCRIPTION
**Pull request by ashwinbhat**
Introduce a new uniform (`u_envLightIntensity`) that allows applications to control the light intensity directly in generated shaders. The uniform is a linear multiplier on environment lights. Applications can control the value of the uniform as needed.

image

### CODE REVIEW

File: source/MaterialXView/RenderPipelineGL.cpp [hunk 1]
```diff
@@ -370,6 +370,9 @@ void GLRenderPipeline::renderFrame(void*, int shadowMapSize, const char* dirLigh
                 float longitudeOffset = (lightRotation / 360.0f) + 0.5f;
                 envMaterial->modifyUniform("longitude/in2", mx::Value::createValue(longitudeOffset));
 
+                // Apply light intensity to the environment shader.
```
Comment (jstone-lucasfilm) [inline source/MaterialXView/RenderPipelineGL.cpp:373]: Do we need a corresponding line to be added to RenderPipelineMetal.mm as well?

https://github.com/AcademySoftwareFoundation/MaterialX/blob/main/source/MaterialXView/RenderPipelineMetal.mm#L491

**Review by jstone-lucasfilm — COMMENTED** [review]
This looks nearly ready to merge, @ashwinbhat, though I had one question about support in the Viewer on MacOS.

File: source/MaterialXView/RenderPipelineGL.cpp (hunk 1 above)
Comment (ashwinbhat) [reply source/MaterialXView/RenderPipelineGL.cpp:373]: image
Thanks @jstone-lucasfilm, I had missed this. I've updated it now, and verified it on MacOS.

**Review by jstone-lucasfilm — APPROVED** [review]
This looks great to me, thanks @ashwinbhat!

**Pull request merged**